src/test/java/org/testcontainers/containers/CephContainerTest.java

```diff
@@ -192,7 +198,7 @@ void testOverrideWaitingFor() {
             assertThat(container.getWaitStrategy()).isInstanceOf(HostPortWaitStrategy.class);
         }
     }
-    // configuringClient {
+    // configuringClient
```

Comment:
aaaactually you can remove these comments cant you .. unless they give some new meaning ?